fera_full.py

```
@@ -1,3 +1,189 @@
+import sys, getopt, global_settings, indexador_fera, setproctitle, fera, utilities_general, global_settings, classes_general
+import os,sqlite3, time, shutil
+from pathlib import Path
+import traceback
+
+
+def start_up_app():
+    sqliteconn = None
+    try:
+        if(len(sys.argv) == 1): 
+            indexador_fera.import_create_toplevel()            
+            if(global_settings.pathdb == None):
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)